src/PHPCR/Util/ValueConverter.php

```diff
@@ -130,11 +130,6 @@ public function convertType($value, $type, $srctype = PropertyType::UNDEFINED)
             && $value instanceof NodeInterface
         ) {
             /** @var $value NodeInterface */
-            // In Jackrabbit a new node cannot be referenced until it has been persisted
-            // See: https://issues.apache.org/jira/browse/JCR-1614
-            if ($value->isNew()) {
-                throw new ValueFormatException('Node ' . $value->getPath() . ' must be persisted before being referenceable');
```

Comment:
Should I do that? Although I don't really know where to put this check :confused: